lib/street_address/us.rb

```diff
@@ -247,7 +245,7 @@ class << self
         (?: (?:#{unit_prefix_numbered_regexp} \W*)
             | (?<unit_prefix> \#)\W*
         )
-        (?<unit> [\w-]+)
+        (?<unit> [\w\/-]+)
```

Comment:
@mtogstad what does this change?

Reply:
This allows a "/" in the "unit" section
e.g. 1213 w mifflin st unit b/101